src/orion/core/worker/trial.py

```diff
@@ -283,11 +283,6 @@ def objective(self):
         """
         return self._fetch_one_result_of_type('objective')
 
-    @property
```

Comment:
This code change is already fixed in the current develop. We might want to remove it to avoid unnecessary merge conflicts?